pkg/ottl/expression_test.go

```diff
@@ -1550,3 +1552,85 @@ func Test_StandardPMapGetter_WrappedError(t *testing.T) {
 	_, ok := err.(TypeError)
 	assert.False(t, ok)
 }
+
+func Test_StandardDurationGetter(t *testing.T) {
```

Comment:
Please also add a `_WrappedError` test, similar to `Test_StandardStringGetter_WrappedError`

Reply:
Added!